internal/payloads/null_convert.go

```diff
@@ -0,0 +1,11 @@
+package payloads
+
+import "database/sql"
+
+func NullConvert(s sql.NullString) *string {
+	str := ""
+	if s.Valid {
+		str = s.String
+	}
+	return &str
+}
```

Comment:
So the idea is to return `nil` when the SQL value is `NULL` not empty string. It prehaps makes sense for string, but once we add integers, we simply cannot use 0 as a value for `NULL`. To be consistent across types and because you use string pointer (not string) in the whole patch, this needs to return `nil` for invalid SQL value.